toolkit/scripts/toolchain/check_manifests.sh

```diff
@@ -8,19 +8,19 @@ TOOLCHAIN_SPEC_LIST_FILE=$1
 SPECS_DIR=$2
 MANIFESTS_DIR=$3
 DIST_TAG=$4
-ARCH=$5
+DISTRO_MACRO=$5
 
 write_rpms_from_spec () {
     # $1 = spec file
     # $2 = file to save to
     spec_dir=$(dirname $1)
-    exclusiveArch=$(rpmspec -q $1 --define="with_check 0" --define="_sourcedir $spec_dir" --define="dist $DIST_TAG" --qf="[%{EXCLUSIVEARCH} ]" --srpm 2>/dev/null)
+    exclusiveArch=$(rpmspec -q $1 --define="with_check 0" --define="_sourcedir $spec_dir" --define="$DIST_TAG" --define="$DISTRO_MACRO" --qf="[%{EXCLUSIVEARCH} ]" --srpm 2>/dev/null)
```

Comment:
`DIST_TAG` only has the value, not the name of the macro.
```suggestion
    exclusiveArch=$(rpmspec -q $1 --define="with_check 0" --define="_sourcedir $spec_dir" --define="dist $DIST_TAG" --define="$DISTRO_MACRO" --qf="[%{EXCLUSIVEARCH} ]" --srpm 2>/dev/null)
```